cdk/index.ts

```diff
@@ -1,19 +1,55 @@
+import * as cognito from "@aws-cdk/aws-cognito";
+import * as ssm from "@aws-cdk/aws-ssm";
 import * as cdk from "@aws-cdk/core";
 import "source-map-support/register";
 import { StaticSite } from "./static-website";
 
+interface StaticSiteProps extends cdk.StackProps {
+  environmentId: string;
+}
+
 class StaticSiteStack extends cdk.Stack {
-  constructor(parent: cdk.App, name: string, props: cdk.StackProps) {
+  constructor(parent: cdk.App, name: string, props: StaticSiteProps) {
     super(parent, name, props);
 
-    new StaticSite(this, "StaticSite");
+    const site = new StaticSite(this, "StaticSite");
+    const poolId = ssm.StringParameter.fromStringParameterName(
+      this,
+      "UserPoolId",
+      `/atat/${props.environmentId}/cognito/userpool/id`
+    ).stringValue;
+    const userPool = cognito.UserPool.fromUserPoolId(this, "UserPool", poolId);
+    userPool.addClient("ApplicationUserPoolClient", {
+      // Validate these validities against any STIG requirements
```

Comment:
The ASD STIG has CAT II vulnerabilities referring to session timeout due to user inactivity but no forced timeout requirement so I think we're good here as is.

FWIW the inactivity limits are 15 minutes for regular users and 10 minutes for privileged users.

Reply:
In some cases I've found it easiest to just go with the 10 minute limit across the board.